docs/Using Price Feeds/historical-price-data.html

```diff
@@ -17,9 +17,19 @@
 
 The most common use case for Data Feeds is to [Get the Latest Price](../get-the-latest-price/). However, <a href="https://github.com/smartcontractkit/chainlink/blob/master/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol" target="_blank" rel="noreferrer, noopener">`AggregatorV3Interface`</a> also exposes functions which can be used to retrieve price data of a previous round ID.
 
-> 📘 Decentralized Data Model
->
-> The [Decentralized Data Model](../architecture-decentralized-model/) describes how Data Feeds are aggregated from many data sources and published on-chain.
+There are two parameters that can cause Chainlink nodes to update:
+
+| Name                   | Description                                                          |
+| ------------------------------- | -------------------------------------------------------------------- |
+|  Deviation Threshold     | Chainlink nodes are monitoring prices of assets off-chain. The deviation of the real-world price of an asset beyond a certain interval triggers all the nodes to update. Right now, most Ethereum data feeds have a 0.5% deviation threshold.                  |
```

Comment:
```suggestion
|  Deviation Threshold     | Chainlink nodes are monitoring prices of assets off-chain. The deviation of the real-world price of an asset beyond a certain interval triggers all the nodes to update.                  |
```
Theoretically this can change and people can find this out by looking at the feeds directly. Let's remove it for now.